### DISCUSSION TITLE
FAQ or documentation

### DISCUSSION THREAD

**Discussion by alchemiker** [AnySoftKeyboard/AnySoftKeyboard#2802] [Ideas]
Hello @menny , at first I would like to thank you for developing and maintaining this great app. I cannot imagine how much time and effort you put into it!

But when I look into the "Issues" here on GitHub and the recensions on Google Play Store, there is one main issue: the settings and options are too confusing. People complain about missing features or undesired behaviour which could be changed in the settings.

No, I won't suggest to re-structure all settings (although this would be great someday), but maybe it would be nice to have a FAQ (Frequent Asked Questions) with instructions for some often-asked configurations, like "suggestions, but without auto-correction" and "add a dictionary of one language to the keyboard from another language" and a section for cool hidden features like "delete whole words" and "add emojis from suggestion bar". Over time it could grow and split into a FAQ and a detailed documentation.

Maybe the wiki here at github would be the right place to start it. Maybe some other users would join in writing, I believe there are also other users who are eager to help but not proficient enough at programming to contribute code to this project.
If you will think later that the FAQ is fine you can add it to ASKs website or link it from inside the app later.
Oops, I wrote so much again...

➡️ **TL;DR: Should I try to write a FAQ or documentation for this project?**

**Comment by menny** [comment]
hello,
I'm really happy to hear you enjoy this app and are ready to contribute back!

I would love some help with documentation, FAQ/Wiki would be great!
I can give you access to edit/create wiki pages, would that be okay?

**Reply by dagelf** [reply]
Isn't there a way we can host it so that we can do pull requests? 

Eg. like [CONTRIBUTING.md](https://github.com/AnySoftKeyboard/AnySoftKeyboard/blob/main/CONTRIBUTING.md) but just FAQ.md or MANUAL.md. And the Wiki page can just provide a link. 

Or even better: A new page on the website: "Manual" or "Documentation". https://github.com/AnySoftKeyboard/AnySoftKeyboard.github.io Much easier for anyone to clone and edit and submit pull requests.

**Reply by menny** [reply]
there is this way to create PRs for the wiki: https://stackoverflow.com/questions/10642928/how-to-pull-request-a-wiki-page-on-github
but, I'm not really sure it makes sense.
You are welcome to suggest any way you feel is the right way (as you suggested, a markdown file in the repo is cool, a page in the website will also work, maybe even better)

**Reply by alchemiker** [reply]
I think both options would be fine. I would tend towards a wiki, because I'm quite unexperienced at things like pull requests, but I could try to learn it. We should chose the option that is the least nuisance to menny. I don't know whether he would have to approve every pull request?
@Jonta I believe I remember that you were interested in writing some documentation, too? In case my memory be misleading I apologize in advance for pinging you for nothing.

**Reply by menny** [reply]
I believe WiKi will be easiest. I think that if I add you to https://github.com/orgs/AnySoftKeyboard/teams/documentors you will be able to edit wiki pages. How is interested?

**Reply by Jonta** [reply]
@alchemiker Sounds like something I'd say
I want action on this one first though: https://github.com/AnySoftKeyboard/AnySoftKeyboard/discussions/3023 :)
